doc-QuickStart_OpenShift_Provider_Guide/topics/image-scanning.adoc

```diff
@@ -0,0 +1,80 @@
+== Configuring Image Scanning
+
+{product-title} manages vulnerability scanning of container images. When an OpenShift provider is added, OpenShift images from the internal registry are discovered. To enable image scanning, perform the following configuration steps:
+
+. Navigate to menu:Compute[Containers > Providers].
+. Select the *OpenShift* provider checkbox.
+. From the *Policy* pull-down menu, click *Manage Policies*.
+. Select the *OpenSCAP profile* checkbox.
+. Click *Save*.
+
+This action will trigger a SmartState analysis, or scan, of all images referenced by the OpenShift provider. The initial scan may take several hours to complete, depending on the number and size of images. OpenShift scanning pods are scheduled for a maximum of three at a time.
```

Comment:
This sentence is a little confusing: "OpenShift scanning pods are scheduled for a maximum of three at a time." Does it mean: "A maximum of three OpenShift pods can be scanned at a time."? Would you mind rewording just a little please?